presto-docs/src/main/sphinx/presto-cpp.rst

```diff
@@ -0,0 +1,55 @@
+**********
+Presto C++
+**********
+
+Note: Presto C++ is in active development. See :doc:`Limitations </presto_cpp/limitations>`.
+
+.. toctree::
+    :maxdepth: 1
+
+    presto_cpp/features
+    presto_cpp/limitations
+    presto_cpp/properties
+
+Overview
+========
+
+Presto C++, sometimes referred to by the development name Prestissimo, is a C++ 
+drop-in replacement for Presto workers based on the `Velox <https://velox-lib.io/>`_ library. It 
+implements the same RESTful endpoints as Java workers using the Proxygen C++ 
+HTTP framework. 
+Because communication with the Java coordinator and across workers is only 
+done using the REST endpoints, Presto C++ does not use JNI and is commonly 
```

Comment:
"and is commonly 
deployed in such a way as to avoid having a JVM process on worker nodes."

-->

does not require a JVM on worker nodes.